### ISSUE TITLE
New typings are missing some functions of the View API related to signals

### ISSUE THREAD

**Issue by baaleze** [vega/vega-typings#10]
I've been using vega for several weeks now in Angular projects and I'm happy to see that an official typings files are being made!

Unfortunately, right now the typings are missing a couple functions which I rely on in order to interact with graphs via signals. These functions are :

```typescript
// in types/runtime/index.d.ts in the class View
addSignalListener(name: string, handler: (name: string, value: any) => void): View;
removeSignalListener(name: string, handler: (name: string, value: any) => void): View;
// only the version with two parameters exists, not this one that allows to get a signal's value
signal(name: string): any;
```

**I understand that the typings are a work in progress right now**. I will be using a old version in the meantime, I just wanted to be notified when these functions are added to the typings.

Thanks for your work!

**Comment by pelotom** [comment]
Hi @baaleze, thanks for the issue. Since you know exactly what’s missing, care to open a PR? 🙂

**Comment by baaleze** [comment]
There you go #11 !

**Comment by pelotom** [comment]
Released the fix in `vega-typings@0.2.10`.

**Comment by domoritz** [comment]
Thank you!
